imports/lib/schemas/ChatNotification.ts

```diff
@@ -17,8 +17,10 @@ const ChatNotificationFields = t.type({
   puzzle: t.string,
   // The hunt in which the puzzle resides.
   hunt: t.string,
-  // The message body. Plain text.
-  text: t.string,
+  // The message body, if chat message v1. Plain text.
+  text: t.union([t.string, t.undefined]),
+  // The message content, if chat message v2.  Actually contains ChatMessageContentType
+  content: t.union([t.UnknownRecord, t.undefined]),
```

Comment:
(Similarly here, I'd prefer if we can make `ChatNotificationType` use the actual `ChatMessageContentType`)

Reply:
Fixed in afc4e4e78da6f8c07dd9e0716cbeecbe68d1f72e